Expecto/Expecto.fs

```diff
@@ -164,8 +164,8 @@ module internal Helpers =
 
     member m.MatchTestsAttributes () =
```

Comment:
this can probably be cut down to something like
```fsharp
    member m.MatchTestsAttributes () =
      (HashSet<_>(), m.GetCustomAttributes true)
      ||> Seq.fold (fun acc t -> acc.Add(t.GetType().FullName)|>ignore;acc)
      |> fun set -> set.Intersect allTestAttributes
      |> Seq.choose matchFocusAttributes
      |> Seq.sortBy fst
      |> Seq.tryFind (fun _ -> true)
```